src/Documentation/DocumentationGenerator/DocumentationWriter.cs

```diff
@@ -145,8 +145,10 @@ public async Task WriteOutput(QsNamespace ns, DocComment docComment)
                 ["title"] = title,
 
                 // docs.ms metadata
-                ["ms.date"] = DateTime.Today.ToString(),
+                ["ms.date"] = DateTime.Today.ToString("MM/dd/yyyy"),
```

Comment:
Does this format need localization?

Reply:
I don't think so. I believe this change is to resolve the Doc warnings like [here](https://github.com/MicrosoftDocs/quantum-api/pull/98), search for "Invalid date format for".

@bradben, please correct me if I'm wrong.

Reply:
Correct. Format can't include the timestamp, only mm/dd/yyyy.